app/src/main/java/com/jerboa/ui/components/post/PostActivity.kt

```diff
@@ -324,8 +343,6 @@ fun PostActivity(
                                                 ),
                                             )
                                         }
-                                        // TODO will need to pass in postlistingsviewmodel
-                                        // for the Home page to also be updated
                                     },
                                     onDownvoteClick = { pv ->
                                         account?.also { acct ->
```

Comment:
> we do this already with the votes.
I'm not sure

Reply:
Yeah i meant that it instantly updates the votes visually not check in the postview itself, we should also make it do the votes using the appState but that's for another PR